.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,48 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Rollup Version (if applicable):
+- Operating System (or Browser):
+- Node Version:
+- NPM Version:
```

Comment:
Rollup has no runtime dependencies so we could remove the npm version, unless they are having TypeScript typing issuese.

Reply:
That's reasonable. Will update.

Reply:
I'd say the Rollup version is *always* relevant. The second line could be:

* Operating System + Node Version (or other Runtime Environment/Browser):

And I agree, the NPM version will probably not be relevant as all NPM specific functionality is usually handled by rollup-plugin-node-resolve.

Reply:
I'm down with removing the NPM version as a requirement, especially given how Rollup is shipped.